examples/hybrid/define_post_processing.jl

```diff
@@ -620,6 +621,49 @@ function paperplots_moist_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
             output_dir * "/bw-vert_intg_water_vapor-day" * string(day) * ".png",
         )
 
+        # Spectrum calculation
+        mass_weight = ones(FT, 1) # only 1 level
+        # First compute 1d spectrum of u-component
+        u_1d_spectrum, freqs = power_spectrum_1d(
+            FT,
+            u,
+            similar(mass_weight),
+            lat,
+            lon,
+            mass_weight,
+        )
+        # Then compute 2d spectrum of u-component
+        u_2d_spectrum, wave_numbers, spherical, mesh_info =
+            power_spectrum_2d(FT, u[:, :, 1, 1], mass_weight) # use first level, 1500m
```

Comment:
which is the index for the first level and what does the other `1` refer to?

Reply:
Here `u` comes from the remapped data from the `NCDataset` above (lines 475 and unpacked on line 490). Its dimensions are `size(u) = (180, 90, 10, 1)` (for default CI pipeline resolution). So the third index is the vertical level, which for this spectrum calculation example I decided to use just the first one, and the fourth one I believe is just the component, so just 1 in this case.